### DISCUSSION TITLE
How to generate platformAppApiKey

### DISCUSSION THREAD

**Discussion by abhimanyu1310** [chatwoot/chatwoot#2542] [Q&A]
Hello. 
I have recently installed Chatwoot to my local server to explore its APIs. Some of the APIs need platformAppApiKey as authentication. Where can I find that in my project. I believe it is different from User token, which I can get in user account itself very easily. But if I want to create user via API then I will need platformAppApiKey but I am unable to find it.

**Comment by sojan-official** [comment]
You can generate them from the rails console.

https://github.com/chatwoot/chatwoot/wiki/Building-on-Top-of-Chatwoot:-Platform-APIs#creating-platform-app

Or you can generate them from super admin pannel 
 - https://www.chatwoot.com/docs/self-hosted/monitoring/super-admin-sidekiq
 
<img width="448" alt="Screenshot 2021-06-29 at 6 25 47 PM" src="https://user-images.githubusercontent.com/73185/123801173-a91a3e80-d907-11eb-9554-f7e8727fb396.png">

- head over to the access tokens section after generating a platform app
